doc/docs.md

```diff
@@ -8113,54 +8113,78 @@ instead use `#!/usr/bin/env -S v -raw-vsh-tmp-prefix tmp run`.
 
 ## Appendix I: Keywords
 
-V has 45 reserved keywords (3 are literals):
+V has 69 reserved keywords:
 
 ```v ignore
+!in
```

Comment:
`!in` is `!`, followed by `in`, there is no need to mention it as a separate case